cruds_adminlte/templates/HttpResponse/403.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Error 403 Forbidden</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <style type="text/css">
+      h1 {
+        font-size: 1200%;
+        font-weight: 900;
+      }
+      #error-page {
+        padding-top: 10%;
+        text-align: center
+      }
+    </style>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+        <div id="error-page" class="col-md-12">
+            <h2>Error 403</h2>
+            <h2>You are not authorized to view this page</h2>
+            <br>
+            <a class="btn btn-primary" href="/">Return to the main page</a>
```

Comment:
It's required to include locate implemententation,  try to use 
{% load i18n %} in the top of file and something like 
{% trans 'Return to the main page' %}